src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -138,6 +138,13 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// </summary>
         public bool StartFromBeginning { get; set; } = false;
 
+        /// <summary>
+        /// Optional.
+        /// Gets or sets the a date and time to initialize the change feed read operation from.
+        /// <remark>It is best to use UTC date / time format to avoid any discrepencies.</remark>
+        /// </summary>
+        public string StartFromTime { get; set; }
```

Comment:
My two biggest concerns with this is time zones and parsing.
1. Does the ChangeFeedProcessor expect UTC? Can we just say that it will be interpreted as UTC no matter what (and then force that?)
2. I mentioned elsewhere, but in the summary including an example of an ISO string would be useful here.

Reply:
#1 The datetime is parsed in C# code. Ideally user can specify in any format that is parsable by C# library. Have added a comment to ideally use ISO 8601.

#2 Added ISO 8601 example in comments

Reply:
@ealsur - thoughts?

Reply:
I'd prefer something like this, which gives an explicit example right in the comment:

`The recommended format is ISO 8601 with the UTC designator. For example: "2021-02-16T14:19:29Z".`

Yes, other formats will work, but this one is guaranteed to work and reduces the need to lookup a separate documentation page and look through all of the options.